https://issues.redhat.com/browse/ACM-25128--adding the bugs that were… (#8471)
open-cluster-management/rhacm-docs: release_notes/acm_errata.adoc

* Delivers updates to one or more product container images.

* Fixes a problem when using multiple workers where a race condition occurred, and metrics were not correctly collected and sent to the Hub Observability stack. When collecting metrics from Prometheus, some of the workers collected the same metrics instead of distinct sets. 
(link:https://issues.redhat.com/browse/ACM-24923[ACM-24923])

* Corrects an issue that caused high memory and CPU usage on the `search-postgres` database when Argo applications were deployed. This high load previously prevented the `search-indexer` pod from synchronizing inventory data. 
(link:https://issues.redhat.com/browse/ACM-23863[ACM-23863])

* Fixes an issue where `ConfigurationPolicies` without annotations caused the controller to crash when the policy add-on was disabled or uninstalled. The controller is now stable throughout the process when all `ConfigurationPolicies` on the managed cluster have an annotation. Manually apply an annotation to any policy that is missing one.

> (link:https://issues.redhat.com/browse/ACM-23253[ACM-23253])

Author: These were clearly stated in the JIRA comments by developers and have not deviated much from that content. Please review line 19 and 22.

> * Delivers updates to one or more product container images.

Author: These other changes just give consistency to spacing in the file, no content changes.

Author: @dockerymick this content was written previously by another writer and we are not gonna change it right now, especially for 2.12, it's been translated as is anyway. We can change the newer content as needed but there is no reason to change these old lines. So let's just leave these as-is for now.

* Fixes the console errors caused by a policy having an invalid structure. 
(link:https://issues.redhat.com/browse/ACM-16238[ACM-16238])

* Stores the correct {ocp-short} version to the database when a cluster gets imported to {acm-short}. 
(link:https://issues.redhat.com/browse/ACM-17553[ACM-17553])

* Allows virtual machine actions on remote clusters with signed certificates. (link:https://issues.redhat.com/browse/ACM-17065[ACM-17065])

> * Preserves the node object even when the `ClusterInstance` gets deleted from the cluster management hub. 

Reviewer: ```suggestion
* Preserves the node object even when the `ClusterInstance` resource gets deleted from the cluster management hub. 
```

(link:https://issues.redhat.com/browse/ACM-17877[ACM-17877])

> * Prevents the same metric from getting reported multiple times and triggering a `PrometheusDuplicateTimestamps` alert by only reporting that metric one time with a count of the number of {ocp-short} templates with violations. 

Reviewer: ```suggestion
* Prevents the same metric from getting reported multiple times and initiating a `PrometheusDuplicateTimestamps` alert by only reporting that metric one time with a count of the number of {ocp-short} templates with violations. 
```

(link:https://issues.redhat.com/browse/ACM-16014[ACM-16014])

> * Stops the metrics collector from degrading the addon status when the collector receives the HTTP 409 error code from the hub cluster, which correctly loads metric samples and rejects duplicates from the payload.  

Reviewer: ```suggestion
* Stops the metrics collector from degrading the add-on status when the collector receives the HTTP 409 error code from the hub cluster, which correctly loads metric samples and rejects duplicates from the payload.  
```
